storyStarterGame.js

```diff
@@ -0,0 +1,321 @@
+       rightPath();
+   } else if (youchoose === undefined){
+       leaveGame();
+   }
+   // if the player choose middle path call middle path function 
+   // need both if statements for middle and right 
+   // need extra if statment for cancel : if youchoose = undefined 
+}
+
+//middlePath
+const middlePath = () => {
+    console.log("Your character finds a beheaded chicken in the middle of the road! \nAaaaaahhhhh, screams are being heard from further." )
+    for (i = 0; i < array.length; i++) {
+        if (array[i] === "middle") {
+            array.splice(i,1)
+        }
+    }
+    let woodsPaths = readLineSync.keyInSelect(array)
+   let youchoose = array[woodsPaths]
+   if (youchoose === "left") {
+    leftPath();
+} else if (youchoose === "right"){
+    rightPath();
+} else if (youchoose === undefined){
+    leaveGame();
+}
+} // if statments left and right and undefined
+
+//rightPath
+const rightPath = () => {
+    console.log("Your character reached a cliff. If they're not careful they will fall!")
+    for (i = 0; i < array.length; i++) {
+        if (array[i] === "right") {
+            array.splice(i,1)
+        }
+    }
+    let woodsPaths = readLineSync.keyInSelect(array)
+   let youchoose = array[woodsPaths]
+   if (youchoose === "middle") {
+    middlePath();
+} else if (youchoose === "left"){
+    leftPath();
+} else if (youchoose === undefined){
+    leaveGame();
+}
+} // if statemt left amnd middle and undefined 
+
+
+// darkRoadStoryLine
+const darkRoadStoryLine = () => {
+    console.log ("Unfortunely the road was so too dark and your character died from a car crashing into them.") 
+    if (readLineSync.keyInYN ("Would you like try another story?")) {
+        darkWoodsStoryLine();
+    } else {
+        leaveGame();
+    }
+
+}
+
+// romanceStory
+const romanceStory = () => {
+     let characterNumRom = readLineSync.question ("How many character you want in this story?") 
+    if (characterNumRom > 3) {
+        tooManyCharRom();
+    } else {
+        perfectNumRom();
+    }
+}
+
+//romSetting
+const romSetting = () => {
+    let romanceSetting = readLineSync.question ("Do you want to start your story in the city or in a home?")
+    if (romanceSetting === "city") {
+        cityDate()
+    } else if (romanceSetting === "home") {
+        stayAtHomeDate()
+    } else {
+        leaveGame()
+    }
+}
+    
+//perfectNum
+const perfectNumRom = () => {
+    console.log ("Perfect!!!")
+    console.log ("You will now have to choose your storyline!")
+    let romSetting = readLineSync.question ("Do you want to start your story in the city or at home?")
+    if (romSetting === "city") {
+        console.clear();
+        cityDate();
+    } else if (romSetting === "home") {
+        stayAtHomeDate();
+    } else {
+        leaveGame();
+    }
+}
+//tooManyCharRom
+const tooManyCharRom = () => {
+    let characterNum = readLineSync.question ("How many character you want in this story?") 
```

Comment:
You should alert the user that they want less than 3 to begin with

Reply:
I will keep that in mind.